specification/storagetasks/resource-manager/Microsoft.StorageTasks/stable/2023-01-01/storagetasks.json

```diff
@@ -460,7 +460,7 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/providers/Microsoft.StorageTasks/previewActions": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.StorageTasks/locations/{location}/previewActions": {
```

Comment:
The change is not a breaking change.
This to change the API from being subscription-wide operation to being location-specific. This is because according to ARM docs. This is because for subscription-wide operation , there is a need for a global endpoint which becomes a Single Point Of Failure for our API
Ref to Arm Doc: https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/proxy-api-reference.md#subscription-wide-reads-and-actions

This API is not released to external customers and is not even in the Manifest
https://armmanifest.visualstudio.com/One/_git/Manifest?path=/MICROSOFT.STORAGETASKS/PROD/MANIFEST.JSON

Reply:
@JeffreyRichter I no ARM concerns for this PR, I think its a brekaing changes concern only (so I'll add the tag)... this is all fine if its unreleased right?

Reply:
Thank you for your review. The changes are not released, and there are no customers consuming this, So it should not break any customers.